### PR TITLE
Background

### LINKED ISSUE #57: [ADD] Ideals, bonds, flaws
A list of SRD compliant data can be found [here ](https://www.5esrd.com/backgrounds/)

### PR DESCRIPTION
**Pull request by HedgehogHolly**
#### Description of pull request
**Background.js**
- added personality, ideal, bond, flaw info
- added function to create random selection for personality traits

**Core.js**
- adding this.personalityTraits to character creation

#### Has it been tested?
 
- [X] Yes
- [ ] No
- [ ] Testing not needed

### CODE REVIEW

File: core.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const names = require("./names.js");
 const feats = require("./feats.js");
 const armors = require("./armor");
 const fs = require("fs");
+const background = require("./background.js");
```
Comment (HedgehogHolly) [inline core.js:9]: It seemed like this was needed? but I have no idea

File: background.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            ideal: utils.returnRandomArrayItem(ideal),
+            bond: utils.returnRandomArrayItem(bond),
+            flaw: utils.returnRandomArrayItem(flaw)
+        };
+
+        return personalityTraits;
+
+    }
+
+    exports.returnRandomPersonality = returnRandomPersonality;
```
Comment (HedgehogHolly) [inline background.js:54]: Not sure if this export is written right?

File: core.js (hunk 1 above)
Comment (KingFruit85) [reply core.js:9]: Yep, so when you want to make access of another files functions you need to include this statement in the file calling the function so it knows where the remote function is stored.

File: background.js (hunk 2 above)
Comment (KingFruit85) [reply background.js:54]: Perfect 👍

**Review by KingFruit85 — APPROVED** [review]
Looks good to me :)

**Comment by KingFruit85** [comment]
How did you link the pull request to the issue ticket btw?

**Pull request merged**